src/DependencyInjection/Configuration.php

```diff
@@ -35,6 +35,7 @@ public function getConfigTreeBuilder()
                             ->thenInvalid('"dc-aware-round-robin" load balancing option require a "dc_options" entry in your configuration')
                         ->end()
                         ->children()
+                            ->booleanNode('persistent_session')->defaultValue(true)->end()
```

Comment:
I think it would be better to name the configuration `persistent_sessions` (with a `s`) to match the name of the Cassandra driver method `withPersistentSessions()`. 
What do you think ?